app/policies/log_policy.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class LogPolicy < TeamDependantPolicy
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def index?
+    @user.is_a?(User::Human) && team.teammember?(@user.id)
+  end
+
+  def show?
+    team.teammember?(@user.id)
```

Comment:
do we have to override default behavior from TeamDependantPolicy here?

Reply:
I think we have overridden this method because we also want to allow `Api users` to access the logs. Should we change that?

Reply:
for now, Api Users should not have access to logs.
their read access should be logged though